<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>login_dialog</class>
    <widget class="QDialog" name="login_dialog">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>760</width>
                <height>680</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Dialog</string>
        </property>
        <widget class="QLineEdit" name="username_input">
            <property name="geometry">
                <rect>
                    <x>220</x>
                    <y>170</y>
                    <width>321</width>
                    <height>41</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <pointsize>10</pointsize>
                </font>
            </property>
            <property name="alignment">
                <set>Qt::AlignCenter</set>
            </property>
            <property name="readOnly">
                <bool>false</bool>
            </property>
            <property name="placeholderText">
                <string>Username</string>
            </property>
        </widget>
        <widget class="QLineEdit" name="password_input">
            <property name="geometry">
                <rect>
                    <x>220</x>
                    <y>280</y>
                    <width>321</width>
                    <height>41</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <pointsize>10</pointsize>
                </font>
            </property>
            <property name="text">
                <string />
            </property>
            <property name="echoMode">
                <enum>QLineEdit::Password</enum>
            </property>
            <property name="alignment">
                <set>Qt::AlignCenter</set>
            </property>
            <property name="placeholderText">
                <string>Password</string>
            </property>
        </widget>
        <widget class="QLabel" name="username_label">
            <property name="geometry">
                <rect>
                    <x>325</x>
                    <y>140</y>
                    <width>110</width>
                    <height>21</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <pointsize>12</pointsize>
                </font>
            </property>
            <property name="text">
                <string>Username</string>
            </property>
            <property name="alignment">
                <set>Qt::AlignCenter</set>
            </property>
        </widget>
        <widget class="QLabel" name="password_label">
            <property name="geometry">
                <rect>
                    <x>320</x>
                    <y>250</y>
                    <width>120</width>
                    <height>21</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <pointsize>12</pointsize>
                </font>
            </property>
            <property name="text">
                <string>Password</string>
            </property>
            <property name="alignment">
                <set>Qt::AlignCenter</set>
            </property>
        </widget>
        <widget class="QPushButton" name="save_button">
            <property name="enabled">
                <bool>false</bool>
            </property>
            <property name="geometry">
                <rect>
                    <x>325</x>
                    <y>520</y>
                    <width>110</width>
                    <height>50</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <pointsize>12</pointsize>
                </font>
            </property>
            <property name="text">
                <string>Save</string>
            </property>
            <property name="default">
                <bool>false</bool>
            </property>
        </widget>
        <widget class="QLineEdit" name="new_password_input">
            <property name="geometry">
                <rect>
                    <x>220</x>
                    <y>410</y>
                    <width>321</width>
                    <height>41</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <pointsize>10</pointsize>
                </font>
            </property>
            <property name="echoMode">
                <enum>QLineEdit::Password</enum>
            </property>
            <property name="alignment">
                <set>Qt::AlignCenter</set>
            </property>
            <property name="placeholderText">
                <string>New Password</string>
            </property>
        </widget>
        <widget class="QLabel" name="new_password_label">
            <property name="geometry">
                <rect>
                    <x>295</x>
                    <y>380</y>
                    <width>171</width>
                    <height>21</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <pointsize>12</pointsize>
                </font>
            </property>
            <property name="text">
                <string>New Password</string>
            </property>
            <property name="alignment">
                <set>Qt::AlignCenter</set>
            </property>
        </widget>
    </widget>
    <resources />
    <connections />
</ui>